runtime/compilers/rillv1/init.go

```diff
@@ -29,7 +29,7 @@ func InitEmpty(ctx context.Context, repo drivers.RepoStore, instanceID, title st
 	if gitignore != "" {
 		gitignore += "\n"
 	}
-	gitignore += ".DS_Store\n\n# Rill\n*.db\n*.db.tmp\n*.db.wal\n.env\n"
+	gitignore += ".DS_Store\n\n# Rill\n*.db\n*.db.tmp\n*.db.wal\n.env\ntmp\n"
```

Comment:
Guess we don't need `*.db` anymore if we put it all in `tmp`?

Reply:
We will need it for backward compatibility ?

Reply:
Yeah this is only wwhen initializing a new project